mlblocks_primitives/keras.Sequential.LSTMTimeSeriesRegressor.json

```diff
@@ -0,0 +1,122 @@
+{
+    "name": "keras.Sequential.LSTMTimeSeriesRegressor",
+    "author": "Ihssan Tinawi <itinawi@mit.edu>",
+    "documentation": "",
+    "description": "This primitive consists of multiple Keras layers that can pass time-series data through an LSTM in order to predict the value at x_{t+1}",
```

Comment:
Since the number of steps to predict can be different from 1 (controlled by dense_units), I would rather say "to predict the `n` next values of the time-series"